com.microsoft.mrtk.graphicstools.unity/Runtime/Shaders/GraphicsToolsStandard.shader

```diff
@@ -25,6 +25,7 @@ Shader "Graphics Tools/Standard"
         _NormalMapScale("Scale", Float) = 1.0
         [Toggle(_EMISSION)] _EnableEmission("Enable Emission", Float) = 0.0
         [HDR]_EmissiveColor("Emissive Color", Color) = (0.0, 0.0, 0.0, 1.0)
+        _EmissiveMap("Emissive Map",2D)="white"{}
```

Comment:
nitpick: missing spaces:
```suggestion
        _EmissiveMap("Emissive Map",2D) = "white" {}
```

Reply:
Could you also bring this change into the [Graphics Tools/Standard Canvas](https://github.com/microsoft/MixedReality-GraphicsTools-Unity/blob/385952c3e0c59b934f7846cb2b815977cb22dcda/com.microsoft.mrtk.graphicstools.unity/Runtime/Shaders/GraphicsToolsStandardCanvas.shader#L27) shader? 

That is the shader used for UnityUI. Simply adding this line should enable the feature for both shaders since they share common include files.